tox.ini

```diff
@@ -2,8 +2,8 @@
 skipsdist = True
 skip_missing_interpreters = True
 envlist =
-    py3{4,5,6,7,8}-test-{api,sdk,example-app,ext-wsgi,ext-http-requests}
-    pypy3-test-{api,sdk,example-app,ext-wsgi,ext-http-requests}
+    py3{4,5,6,7,8}-test-{api,sdk,example-app,ext-wsgi,ext-http-requests,ext-jaeger}
+    pypy3-test-{api,sdk,example-app,ext-wsgi,ext-http-requests,ext-jaeger}
     lint
     py37-{mypy,mypyinstalled}
```

Comment:
Since you use type annotations, it would make sense to also add this package to the mypy environment.

Reply:
I'm not totally sure how to do it. Could you please provide an example?

Reply:
Just look at the "mypy:" prefixed lines later in the tox.ini and add a corresponding line for the exporter (though that will almost certainly turn up lots of errors, so maybe do that in a separate PR. Cf. #201).